stream-chat-android-offline/src/test/java/io/getstream/chat/android/livedata/usecase/KeystrokeImplTest.kt

```diff
@@ -3,17 +3,16 @@ package io.getstream.chat.android.livedata.usecase
 import androidx.test.ext.junit.runners.AndroidJUnit4
 import com.google.common.truth.Truth
 import io.getstream.chat.android.livedata.BaseConnectedIntegrationTest
-import kotlinx.coroutines.runBlocking
-import kotlinx.coroutines.yield
+import kotlinx.coroutines.test.runBlockingTest
 import org.junit.Test
 import org.junit.runner.RunWith
 
 @RunWith(AndroidJUnit4::class)
 internal class KeystrokeImplTest : BaseConnectedIntegrationTest() {
 
     @Test
-    fun keystroke() = runBlocking {
-        yield()
+    fun keystroke() = runBlockingTest {
+        advanceUntilIdle()
```

Comment:
We should place usages of `advanceUntilIdle` to where we expect data propagation to occur (in this test, between keystrokes and assertions). It's a coincidence that placing it at the start of the function works, and we shouldn't rely on that.